src/NerdMinerV2.ino.cpp

```diff
@@ -96,7 +101,11 @@ void setup()
   /******** SHOW LED INIT STATUS (devices without screen) *****/
   mMonitor.NerdStatus = NM_waitingConfig;
   doLedStuff(0);
-  
+
+#ifdef NERDMINER_T_HMI
```

Comment:
Here too, could the condition can be another thing more generic like  "TOUCH_ENABLE" ?

Reply:
yeah,  good one.